Test/comp/NestedArrays.dfy

```diff
@@ -10,11 +10,7 @@
  * fixed, Java is omitted from this file.
  *
  * Also, note, these tests mostly check that auto-initialized variables of array
- * types get proper values. Without --relax-definite-assignment (which is
- * our new, planned default), there will still be some way to ask the compiler
- * to generate these values (a plan is to change ":= *" to be evidence for a
- * definite assignments). At that time, the functionality tested in this
- * file will still be needed in the compilers.
+ * types get proper values.
```

Comment:
I can't believe we have no documentation for `:= *`
Otherwise, I would have asked you to add something to the doc about that. If not you, @davidcok , would you be able to add documentation on `:= *`?

Reply:
Good point!

Reply:
There is (at least now) documentation on havoc assignments. It presumes that any new values satisfies whatever non-null or subset constraints the type of the LHS has.